tests/CMakeLists.txt

```diff
@@ -189,13 +189,14 @@ set(adventure_SRCS
         )
 set(TestAdventure_SRCS testadventure.cpp testadventure.h)
 add_executable(TestAdventure ${TestAdventure_SRCS} ${adventure_SRCS})
+add_dependencies(TestAdventure glm)
```

Comment:
So just to double check, *all* tests should add this glm dependency explicitly?

Reply:
GmcpMessage seems to be dependent on src/global/bits.h

We should probably convert our core business logic into a lib that is used by the executable and tests